bindings/c/include/vw/experimental/types.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+#pragma once
+
+// uint32_t, uint64_t, etc
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+  typedef uint32_t VWStatus;
+  static const VWStatus VW_success = 0;
```

Comment:
The definitions in `error_data.h` are all lower case, so we are forced to make them lower case here. I think it might be work using upper case names though?

Reply:
are ints preferable over enums here?

Reply:
I don't have a strong preference either way. Keep in mind they are c enums here so the typing isn't really there